sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentAnalysisClientBuilderExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure;
+using Azure.AI.FormRecognizer.DocumentAnalysis;
+using Azure.Core.Extensions;
+
+namespace Microsoft.Extensions.Azure
+{
+    /// <summary>
+    /// Extension methods to add the <see cref="DocumentAnalysisClient"/> to clients builder.
+    /// </summary>
+    public static class DocumentAnalysisClientBuilderExtensions
```

Comment:
Where are these extensions used?

Reply:
They're used for integration with ASP.NET Core. That's described in our [guidelines](https://azure.github.io/azure-sdk/dotnet_implementation.html#integration-with-aspnet-core).